web-local/src/lib/http-request-queue/HttpRequestQueue.ts

```diff
@@ -15,21 +15,29 @@ import {
   fetchWrapper,
   FetchWrapperOptions,
 } from "@rilldata/web-local/lib/util/fetchWrapper";
-import { waitUntil } from "@rilldata/web-local/lib/util/waitUtils";
 
 export const UrlExtractorRegex =
   /v1\/instances\/[\w-]*\/(metrics-views|queries)\/([\w-]*)\/([\w-]*)\/(?:([\w-]*)(?:\/|$))?/;
 
 // intentionally 1 less than max to allow for non profiling query calls
-const QueryQueueSize = 5;
+let QueryQueueSize = 20;
+try {
+  if (
+    window.location.protocol === "https:" &&
+    window.location.host !== "localhost"
```

Comment:
should this be https OR localhost instead of AND ?

Reply:
Updated